infrastructure/ansible/roles/certbot/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+- name: install software properties common
+  become: true
+  apt:
+    state: present
+    name: software-properties-common
+  
+- name: install certbot apt-repositories
+  become: true
+  apt_repository:
+    state: present
+    repo: "{{ item }}"
+  loop:
+    - "deb http://archive.ubuntu.com/ubuntu/ bionic universe"
+    - "deb http://archive.ubuntu.com/ubuntu/ bionic-updates universe"
+    - "deb http://security.ubuntu.com/ubuntu/ bionic-security universe"
+    - "ppa:certbot/certbot"
+
+- name: install certbot and python for certbot on nginx
+  become: true
+  apt:
+    update_cache: yes
+    name:
+      - certbot
+      - python-certbot-nginx
+
+- name: look for certbot generated key file
+  become: true
+  stat:
+    path: /etc/letsencrypt/live/prerelease.triplea-game.org/cert.pem
```

Comment:
I assume this path shouldn't be hardcoded?

Reply:
Indeed, good catch, updated: 828cce124